.github/ISSUE_TEMPLATE/questions.yml

```diff
@@ -0,0 +1,15 @@
+name: Questions
+description: Ask any unanswered question 
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please make sure that your question is not covered in the [documentation](https://openroad.readthedocs.io/en/latest/) before filling the form
```

Comment:
Can we also add a link to this page: https://github.com/The-OpenROAD-Project/OpenROAD/discussions

Reply:
I think it is better to not have a form and on https://github.com/The-OpenROAD-Project/OpenROAD/issues/new/choose just say tell people to use the dicussion page (w/link) for questions.  The other form is fine.

Reply:
@maliberty where exactly do I tell people to use the discussion page?

Reply:
In the page I linked above